content/200-concepts/100-components/02-prisma-client/051-working-with-fields/100-working-with-json-fields.mdx

```diff
@@ -874,6 +878,12 @@ To differentiate between these possibilities, we've introduced three _null enums
 - `DbNull`: Selects the `NULL` value in the database.
 - `AnyNull`: Selects both `null` JSON values and `NULL` database values.
 
+<Admonition type="info">
+
+From v4.0.0, `JsonNull`, `DbNull`, and `AnyNull` are [JavaScript symbols](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol). Before v4.0.0, they were strings.
```

Comment:
Just a heads up that this may change. We could abstract this out and just say "constants" or wait on a decision from @aqrln on his approach/

Relevant discussion: https://prisma-company.slack.com/archives/C02FZENHD4N/p1655923522177859?thread_ts=1655821379.664189&cid=C02FZENHD4N

Reply:
Yep, don't worry, I have my eye on this. I'll keep in touch with @aqrln  about this.